src/makeCalculator.js

```diff
@@ -4,7 +4,27 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+  calculator.add = (number) => (calculator.result += number);
+  calculator.subtract = (number) => (calculator.result -= number);
+  calculator.multiply = (number) => (calculator.result *= number);
+  calculator.divide = (number) => (calculator.result /= number);
+
+  calculator.operate = (modification, number) => {
+    modification(number);
+
+    return calculator;
+  };
```

Comment:
The `operate` method currently takes a function `modification` and a `number` as arguments. While this provides flexibility, it might be more user-friendly to accept operation names as strings (e.g., 'add', 'subtract') and perform the corresponding operation internally. This would make the method easier to use for those unfamiliar with passing functions as arguments.